### PR TITLE
make console examples both Linux/Mac and Windows

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: workshop/content/docs/publishing/ogcapi-features.md [hunk 1]
```diff
@@ -92,23 +92,45 @@ This exercise is optional, if time permits, or you can perform this after worksh
...
-cd docker/elastic
+=== "Linux/Mac"
 
-docker compose up
-```
-</div>
+    <div class="termy">
+    ```bash
+    cd docker/elastic
+
```
Comment (doublebyte1) [inline workshop/content/docs/publishing/ogcapi-features.md:120]: In the other terminals, we don't have a line between `cd` and `docker compose`.  See: https://github.com/geopython/diving-into-pygeoapi/pull/185/files#diff-498dbf04e1849ba7fa83e21c99d9d76b1361b5f410c2b1b887eaa1038180d1dcR39

File: workshop/content/docs/publishing/ogcapi-records.md [hunk 2]
```diff
@@ -69,40 +69,77 @@ develop your own custom pygeoapi plugin, or convert your metadata to OGC API - R
 
     If you do not have Python installed, consider running this exercise in a Docker container. See the [Setup Chapter](../setup.md#using-docker-for-python-clients).
 
+    === "Linux/Mac"
```
Comment (doublebyte1) [inline workshop/content/docs/publishing/ogcapi-records.md:72]: I think the indentation needs to be:

### Sample ISO 19139 to TinyDBCatalogue loader

It is possible to load more example ISO19139 metadata in a TinyDB database with [the following script](https://github.com/geopython/pygeoapi/blob/master/tests/load_tinydb_records.py) ([raw](https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py)):

=== "Linux/Mac"

    <div class="termy">
    ```bash
    cd workshop/exercises/data/tartu/metadata
    curl -O https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py
    python3 load_tinydb_records.py xml catalogue.tinydb
    ```
    </div>

=== "Windows (PowerShell)"

    <div class="termy">
    ```bash
    cd workshop/exercises/data/tartu/metadata
    curl https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_tinydb_records.py
    python3 load_tinydb_records.py xml catalogue.tinydb
    ```
    </div>

**Review by doublebyte1 — CHANGES_REQUESTED** [review]
Great improvement! Small changes requested

**Comment by tomkralidis** [comment]
Thanks @doublebyte1.  Fixed for review.

**Review by doublebyte1 — APPROVED** [review]

**Pull request merged**